### PR TITLE
Updates to Readme/Overview section 

### PR DESCRIPTION
**Pull request by devsatishm**
### Description of the Change

- Updates to Readme section
- Added GitHub badges

### Possible Drawbacks

NA

### Applicable Issues

NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
-* **Other Features** - Other useful features/integrations like SCM (Gitlab/Bitbucket), monitoring (CloudWatch), logging (ElasticSearch), authentication (Cognito), code quality metrics (SonarQube) comes with Jazz by default.
-* **Extensions** - Jazz is designed to integrate and work well with other systems that your enterprise needs. You can check out our optional extensions like Slack, Splunk etc. Centralized configuration helps Jazz admins to easily enable/disable these features as per their needs.
+* **Services** - As of today, Jazz can help build functions, APIs and static websites. Fully customizable template-based design makes it easy for admins to define new ones and expose them as services to the developers. Support for [AWS SAM](https://github.com/awslabs/serverless-application-model) & [Serverless Framework](https://github.com/serverless/serverless) templates will be coming soon - this will allow developers to define their serverless applicatinos using these commonly used frameworks!
```
Comment (pspani) [inline README.md]: spelling: applications

File: README.md [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+* **Deployment Targets** - Currently, Jazz can deploy to AWS serverless services including Lambda, API Gateway, S3, CloudFront, DynamoDB, SQS, Kinesis etc. We plan to support related services in Azure and GCP in the near future.
+* **CI/CD** - Jazz comes with CI/CD by default. It creates a code repository for each service, assigns appropriate permissions and adds a web hook to trigger build/deployment workflows whenever it sees a commit. We leverage [Jenkins](https://github.com/jenkinsci/jenkins) open source for build process and [Serverless Framework](https://github.com/serverless/serverless) for deploying these services.
```
Comment (pspani) [inline README.md]: for the build process

File: README.md [hunk 3]
```diff
@@ -1,23 +1,30 @@
...
+* **Extensions** - Jazz is designed to integrate and work well with other systems that your enterprise needs. You can check out our optional extensions like Slack, Splunk, Apigee etc. Centralized configuration helps Jazz admins to easily enable/disable these features as per their needs.
+* **Multiple Accounts & Regions** - Jazz's multi account/region extension allows you to add support for multiple accounts & regions to an existing Jazz stack. Developers can simply select the target account & region from the UI to deploy their services. Admins can add/remove new accounts/regions to the stack with ease.
+* **Other Features** - Jazz comes with many useful integrations like SCM (Gitlab/Bitbucket), monitoring (CloudWatch), logging (ElasticSearch), authentication (Cognito), access controls, code quality metrics (SonarQube) by default.
+* **Utilities** - Jazz comes many utilties that allows admins to monitor, govern and report the usage of serverless services through Jazz environment.
```
Comment (pspani) [inline README.md]: Jazz comes with many utilities

File: README.md (hunk 1 above)
Comment (devsatishm) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (devsatishm) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (devsatishm) [reply README.md]: Fixed.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,30 @@
...
+![GitHub release](https://img.shields.io/github/release/tmobile/jazz.svg)
 [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/TMO-OSS/Jazz)
 [![Slack Chat](https://img.shields.io/badge/Chat-Slack-ff69b4.svg)](https://tmo-oss-getinvite.herokuapp.com/)
 
 **Seamlessly build, deploy & manage cloud-native serverless applications!**
 
 ## Introduction
 
-Jazz, a serverless platform, accelerates adoption of serverless technology within your enterprise. Jazz comes with a beautiful UI that lets developers quickly create serverless applications with a click of a button. Its modular design makes it easy to add new integrations.
+Jazz, a serverless development platform, accelerates adoption of serverless technology within your enterprise. Jazz comes with a beautiful UI that lets developers quickly create serverless applications with a click of a button. Its modular design makes it easy to add new integrations that your enterprise needs.
```
Comment (raghits) [inline README.md]: Should we call it beautiful UI? Developers want Functional UI.

File: README.md [hunk 5]
```diff
@@ -1,23 +1,30 @@
...
+* **Extensions** - Jazz is designed to integrate and work well with other systems that your enterprise needs. You can check out our optional extensions like Slack, Splunk, Apigee etc. Centralized configuration helps Jazz admins to easily enable/disable these features as per their needs.
+* **Multiple Accounts & Regions** - Jazz's multi account/region extension allows you to add support for multiple accounts & regions to an existing Jazz stack. Developers can simply select the target account & region from the UI to deploy their services. Admins can add/remove new accounts/regions to the stack with ease.
+* **Other Features** - Jazz comes with many useful integrations like SCM (Gitlab/Bitbucket), monitoring (CloudWatch), logging (ElasticSearch), authentication (Cognito), access controls, code quality metrics (SonarQube) by default.
+* **Utilities** - Jazz comes many utilties that allow admins to monitor, govern and report the usage of serverless services through Jazz environment.
```
Comment (raghits) [inline README.md]: Should read: Jazz comes with many utilities (typo and missing with)

File: README.md (hunk 4 above)
Comment (devsatishm) [reply README.md]: I want `Beautiful` & `Functional` UI :)

File: README.md (hunk 5 above)
Comment (devsatishm) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (raghits) [reply README.md]: :)

**Pull request merged**